packages/inquirer-common/src/prompts/utility.ts

```diff
@@ -95,27 +95,60 @@ export function ui5VersionsGrouped(
 
     const versionChoices = [...maintChoices, ...notMaintChoices];
     if (defaultChoice) {
-        versionChoices.unshift(defaultChoice);
+        // adds or moves default choice to the top of the list
```

Comment:
Is this comment accurate, we dont appear to be moving anything but unshifting the found version.

Reply:
Sorry the comment was written for a previous implementation but wasn't edited. 
Yes relying on the default method to work